app/src/main/java/app/revanced/manager/ui/screen/settings/AboutSettingsScreen.kt

```diff
@@ -132,9 +161,10 @@ fun AboutSettingsScreen(
                 maxItemsInEachRow = 2,
                 horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally)
             ) {
-                filledButton.forEach { (icon, text, onClick) ->
+                filledButtons.forEach { (icon, text, onClick) ->
                     FilledTonalButton(
-                        onClick = onClick
+                        onClick = onClick,
+                        modifier = Modifier.defaultMinSize(minWidth = 130.dp),
```

Comment:
Maybe better to find a solution instead of arbitrarily choose a width, as an entry being added in the API that has slightly more characters can cause this to fail